app/src/main/java/com/example/criminalintent/presentation/crime_detail/CrimeFragment.kt

```diff
@@ -50,6 +54,44 @@ class CrimeFragment : Fragment(), FragmentResultListener {
         super.onViewCreated(view, savedInstanceState)
         childFragmentManager.setFragmentResultListener(DIALOG_DATE, viewLifecycleOwner, this)
         observeData()
+        crime = Crime()
+        onBackPressed()
+    }
+
+    private fun onBackPressed() {
```

Comment:
Naming starting with `on` represent functions that are called exactly when something happens. 
E.g here, I would expect this to be called when the back button is pressed.
Please find an appropriate name based on the function's code. Hint: you may start with add or set